package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "test-porject",
+  "name": "test-project",
```

Comment:
We can actually start calling it simple-auth, the name was just the test name I used when first trying nestJS

Reply:
Yep, i will also update the other fields from package.json :+1: